src/citrine/informatics/workflows.py

```diff
@@ -162,3 +163,67 @@ def executions(self) -> WorkflowExecutionCollection:
         if getattr(self, 'project_id', None) is None:
             raise AttributeError('Cannot initialize execution without project reference!')
         return WorkflowExecutionCollection(self.project_id, self.uid, self.session)
+
+
+class PredictorEvaluationWorkflow(Resource['PredictorEvaluationWorkflow'], Workflow):
```

Comment:
@maxhutch I relocated this workflow to the informatics package. Is there reason you didn't include this class with the other workflows in your original PR?

Reply:
We just have two patterns -- one where the resources are in `resources` and another where they are in `informatics`.  I'm not opposed to putting it in `informatics` but it is different enough from the other workflows that I'd suggest keeping it in a separate file.